release/pkg/generate_spec_test.go

```diff
@@ -0,0 +1,25 @@
+package pkg
+
+import (
+	"testing"
+)
+
+func TestGenerateBundleManifest(t *testing.T) {
+	testCases := []struct {
+		testName           string
+		actualBundleFile   string
+		expectedBundleFile string
+	}{
+		{
+			testName:           "Default dev-release dry-run",
+			actualBundleFile:   "../downloaded-artifacts/dry-run-bundle-release.yaml",
+			expectedBundleFile: "testdata/bundle-release.yaml",
+		},
+	}
+
+	for _, tt := range testCases {
+		t.Run(tt.testName, func(t *testing.T) {
+			CheckFilesEquals(t, tt.actualBundleFile, tt.expectedBundleFile)
```

Comment:
I'm missing something, how is `../downloaded-artifacts/dry-run-bundle-release.yaml` being generated?
I was expecting it to be generated from the test